cpp/src_prims/score/scores.h

```diff
@@ -78,6 +78,122 @@ namespace MLCommon {
 
             return 1.0 - sse/ssto;
         }
+
```

Comment:
This is more of a question for @cjnolet.

I now notice that we have `score/scores.h` and `metrics/` folder for implementing scores or metrics. Maybe we should just consolidate everything under `Metrics` namespace?

Reply:
This had originally been namespaces under metrics for both the prims and algorithms layers. I recall the prims namespace being renamed to scores at one point but am not completely sure of the reasoning. Tagging @dantegd.

Reply:
I think it's nice to keep everything under `metrics` namespace just like [scikit-learn](https://scikit-learn.org/0.20/modules/classes.html#module-sklearn.metrics). Any objections to this @dantegd @cjnolet @JohnZed ?

Reply:
No objections from me. @dantegd May have been acting on a conversation we had where I pointed out that not all the metrics in the sklearn metrics package were true metrics (Eg symmetric, follows triangle inequality) and that may have originally prompted the renaming. 

I’m definitely more partial to following sklearn’s naming here, though.

Reply:
@teju85 Just checking in to see if you'd like us to make any of the namespace changes etc. for this PR, or help with a separate one. 
Our `accuracy_score` and all the `regression_metrics` are in `scores.h` and have unit-tests.